### PR TITLE
5. Adding Recommendation Updater Service [KRUIZE-VPA Integration]

### PR DESCRIPTION
**Pull request by shekhar316**
## Description

This PR introduces a service for updating deployments for `auto` or `recreate` mode experiments.

### Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Docs update
- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
- [ ] Requires DB changes

## How has this been tested?
Tested on both the ResourceHub cluster.

Test Configuration

Kubernetes clusters tested on: ResourceHub (OpenShift)

## Checklist :dart:

- [x] Followed coding guidelines
- [x] Comments added
- [ ] Dependent changes merged
- [ ] Documentation updated
- [ ] Tests added or updated

### CODE REVIEW

**Review by bharathappali — APPROVED** [review]
LGTM

**Comment by dinogun** [comment]
@shekhar316 Can you please add documentation for the "auto" and "recreate" modes in the design folder. Currently we have an docs for Monitoring and Experiment modes and we can create a common doc in UpdateModes.md

File: src/main/java/com/autotune/database/dao/ExperimentDAOImpl.java [hunk 1]
```diff
@@ -732,6 +732,29 @@ public List<KruizeExperimentEntry> loadAllExperiments() throws Exception {
         return entries;
     }
 
+    @Override
+    public List<KruizeLMExperimentEntry> loadAllLMExperiments() throws Exception {
+        //todo load only experimentStatus=inprogress , playback may not require completed experiments
+        List<KruizeLMExperimentEntry> entries = null;
+        String statusValue = "failure";
+        Timer.Sample timerLoadAllExp = Timer.start(MetricsConfig.meterRegistry());
```
Comment (dinogun) [inline src/main/java/com/autotune/database/dao/ExperimentDAOImpl.java:740]: Is this a new timer that is being added? Can you please update this [file](https://github.com/kruize/autotune/blob/master/design/KruizePromQL.md) with the details on what this means

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**